Sources/Sonar/APIs/AppleRadarRouter.swift

```diff
@@ -59,7 +60,7 @@ enum AppleRadarRouter {
                     "configurationSplitValue": radar.configuration,
                     "workAroundText": "",
                     "descriptionText": radar.body,
-                    "problemAreaTypeCode": radar.area.map { String($0.appleIdentifier) } ?? "",
+                    "problemAreaTypeCode": (radar.area.map { String($0.appleIdentifier) } ?? "") as Any,
```

Comment:
This is fixed in 8.3 beta 4, I think I'll just undo this work around and let the error be in master.